src/helpers/AppRoutes.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
+import MasterLayout from "../layout/MasterLayout";
+import Home from "../posts/components/Homepage";
+import Friends from "../friends/components/Friends";
+import Stories from "../stories/components/Stories";
+
+const AppRoutes: React.FC = () => {
+  return (
+    <MasterLayout>
```

Comment:
please create a new route `\` with this layout, and replace `{children}` in MainLayout.tsx with `<Outlet />`:

```
<Routes>
        <Route path="/" element={<MainLayout />}>
          <Route index element={<Home />} />
          <Route path="/friends" element={<Friends />} />
          <Route path="/posts" element={<Posts />} />
          <Route path="/events" element={<div>Events Page</div>} />
          <Route path="/videos" element={<div>Watch Videos Page</div>} />
          <Route path="/files" element={<div>Files Page</div>} />
        </Route>
      </Routes>
```